app/reducer.js

```diff
@@ -691,32 +707,32 @@ export default function reduce(state, action) {
             if (allActiveEffects[i + 1]) { // and there's one after it
               allActiveEffects[i - 1].node.connect(allActiveEffects[i + 1].node);
             } else { // there's an effect before it, but not after it
-              allActiveEffects[i - 1].node.connect(state.audioContext.destination);
-            }
+            allActiveEffects[i - 1].node.connect(state.audioContext.destination);
+          }
           } else { // there's no effect before it
-            if (allActiveEffects[i + 1]) {
-              state.masterOut.connect(allActiveEffects[i + 1].node);
+          if (allActiveEffects[i + 1]) {
+            state.masterOut.connect(allActiveEffects[i + 1].node);
             } else { // there are no effects left once it's been removed
-              state.masterOut.connect(state.audioContext.destination);
-            }
+            state.masterOut.connect(state.audioContext.destination);
           }
         }
       }
-      allActiveEffects = allActiveEffects.filter((effect) => effect.name !== 'to be deleted');
-      return Object.assign({}, state, {activeEffects: allActiveEffects});
     }
-    case 'NAVIGATE_ROOM': {
-      let room = action.room;
-      state.socket.emit('room', { joinRoom: room,  leaveRoom: state.currentRoom});
+    allActiveEffects = allActiveEffects.filter((effect) => effect.name !== 'to be deleted');
+    return Object.assign({}, state, {activeEffects: allActiveEffects});
+  }
+  case 'NAVIGATE_ROOM': {
+    let room = action.room;
+    state.socket.emit('room', { joinRoom: room,  leaveRoom: state.currentRoom});
 
-      return Object.assign({}, state, {currentRoom: room});
-    }
-    case 'UPDATE_ACTIVE_ROOMS': {
-      return Object.assign({}, state, {activeRooms: action.allActiveRooms});
-    }
-    default: {
-      console.error('Reducer Error: ', action);
-      return Object.assign({}, state);
-    }
+    return Object.assign({}, state, {currentRoom: room});
+  }
+  case 'UPDATE_ACTIVE_ROOMS': {
+    return Object.assign({}, state, {activeRooms: action.allActiveRooms});
   }
+  default: {
+    console.error('Reducer Error: ', action);
+    return Object.assign({}, state);
+  }
+}
 }
```

Comment:
The bottom part of this file already had the right indentation, I'd remove these edits from your commit.